nixos/modules/security/pam.nix

```diff
@@ -943,6 +943,11 @@ let
       value.source = pkgs.writeText "${name}.pam" service.text;
     };
 
+  sudoConfigForSSHAgentAuth = optionalString config.security.pam.enableSSHAgentAuth ''
```

Comment:
add "optional" to variable name, since it's not immediatly clear it's an optionalString value.